### DISCUSSION TITLE
qute-bitwarden not working

### DISCUSSION THREAD

**Discussion by enrico223** [qutebrowser/qutebrowser#7648] [Q&A]
**Version info**:
```
qutebrowser v2.5.4
Git commit: 
Backend: QtWebEngine 5.15.13, based on Chromium 87.0.4280.144
Qt: 5.15.8

CPython: 3.10.10
PyQt: 5.15.9
```

**Does the bug happen if you start with `--temp-basedir`?**:
It doesn't. 

**Description**

**How to reproduce**
Download `qutebrowser` and the necessary dependencies for the plugin. Follow the instruction at the beginning of the plugin file. Try. 

When I do `:spawn --userscript qute-bitwarden`, the plugin does not work. 

Output of `process`: 
```
Traceback (most recent call last):
  File "/usr/share/qutebrowser/userscripts/qute-bitwarden", line 307, in <module>
    sys.exit(main(arguments))
  File "/usr/share/qutebrowser/userscripts/qute-bitwarden", line 221, in main
    pass_(
  File "/usr/share/qutebrowser/userscripts/qute-bitwarden", line 148, in pass_
    process = subprocess.run(
  File "/usr/lib/python3.10/subprocess.py", line 503, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/lib/python3.10/subprocess.py", line 971, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.10/subprocess.py", line 1847, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'bw'
```

**Comment by The-Compiler** [comment]
I converted this to a discussion, since this is a problem with your setup, and not with qutebrowser.

Can you please show the output of:

- `which bw` in a terminal
- `:debug-pyeval os.environ['PATH']` in qutebrowser when launched normally
- The same when launched so that it works (i.e. via `--temp-basedir`)

Also, how do you usually launch qutebrowser?

**Reply by enrico223** [reply]
Thanks to your input, I traced down the problem. The executable for `bw` was in `.local/bin`. I moved it in `/usr/local/bin/` and now it works no problem.
